README.md

```diff
@@ -99,6 +105,50 @@ from the Debug > Theme menu. There is not yet a mechanism for including
 custom themes.
 
 
+## CLI
+
+Xi-Mac includes a CLI for opening files directly from the command line.
+
+### Installing
+
+**Through XiEditor:**
+
+1. Install XiEditor
+2. Open XiEditor
+3. XiEditor > Install Command Line Tool
+
+
+**Manually:**
```

Comment:
I added it because I relied on it a lot during development for debugging, but if it makes installation too murky I'll delete it. A note saying that it is not the recommended method could be added.

Reply:
Yea, I'm just not sure it's ever going to be useful to the general reader?

Reply:
Oh, I'll remove it when I fix all the other readme things.